content/exporters/supported-exporters/Node.js/Instana.md

```diff
@@ -33,10 +33,9 @@ To use Instana as your exporter, first ensure that you have an [Instana agent ru
 Now let's use the Instana exporter:
 
 ```js
-var tracing = require('@opencensus/nodejs');
-var instana = require('@opencensus/exporter-instana');
-
-var exporter = new instana.InstanaTraceExporter();
+const tracing = require('@opencensus/nodejs');
+const { InstanaTraceExporter }= require('@opencensus/exporter-instana');
```

Comment:
This seems spurious, perhaps just keep the `const` change and then as it was before i.e.
```nodejs
const instana
```

Reply:
I personally think the destructuring import is fine as that is a fairly common pattern in JS and then it saves you referencing it via the namespace below (i.e. you don't need to say `const exporter = new instana.InstanaTraceExporter();` you can just say `const exporter = new InstanaTraceExporter();`)